win_images/win-server-wsl.yml

```diff
@@ -34,7 +34,11 @@ builders:
     security_group_id: "sg-042c75677872ef81c"
     ami_name: &ami_name '{{build_name}}-c{{user `IMG_SFX`}}'
     ami_description: 'Built in https://cirrus-ci.com/task/{{user `CIRRUS_TASK_ID`}}'
-
+    launch_block_device_mappings:
+      - device_name: '/dev/sda1'
+        volume_size: 200
+        volume_type: 'gp2'
+        delete_on_termination: true
```

Comment:
This LGTM, though from what I've seen in Mac-land, default `gp2` and `gp3` may be "too slow" for podman-machine testing.  IIRC, they only provision 150 IOPS.  Going up to 'gp3' + 600 IOPS is still not great, but it's a big improvement for a small increase in cost.

If you have a chance to re-push/build, would you mind switching to `gp3` and setting `iops: 600`?